### DISCUSSION TITLE
Creating SOC accounts when using a Standalone Fleet Node

### DISCUSSION THREAD

**Discussion by cyberkimh** [Security-Onion-Solutions/securityonion#6405] [Unsupported Versions] [answered]
Hi, I was wondering if anyone has advice for creating **SOC user accounts** when using a S**tandalone Fleet Node** in your distribution?

We have created SOC user accounts as normal and enabled them on The Hive and Fleet, but we can't log into Fleet with the account.
The original account we created at deployment works, but not the accounts created via terminal using _sudo so-user-add tom@example.com_

We've ran this command _sudo so-fleet-user-enable $youruser 1_ on the manager and the Fleet standalone node, but still the accounts won't log in.

I wondered if there's some other commands we're missing that need to be ran on the Fleet standalone node, or a file that needs the Fleet standalone info adding to so that the accounts sync up.

I'd be grateful for any advice thanks.

**Comment by Sahale** [comment, marked as answer]
Hi.
As far as I know, Fleet has "own" users in SO and they cannot be synced with SOC. New users can be created via web-interface `Settings` > `Users`.

**Reply by cyberkimh** [reply]
It does but that seems to require SMTP being set up which we really don't want to do.

**Reply by Sahale** [reply]
Actually, no. I've just checked it: created user in web interface and then successfully logged in. Everything works without mail setup. SO 2.3.90

**Reply by cyberkimh** [reply]
Ah we were still on 2.3.80 and there was no user account option. Just upgraded and it's worked. Thanks for the tip.
